src/pages/index.js

```diff
@@ -307,50 +307,59 @@ const ProductsSection = props => {
   return (
     <>
       <StyledSectionFlex style={{ paddingBottom: '0px' }}>
-        <StyledGoal style={{ width: '100%', maxWidth: '450px' }}>
+        <StyledGoal style={{ width: '100%', maxWidth: '650px' }}>
           <h1>In Bloom</h1>
           <p>Check out some of these sweet applications and experimental products being worked on by 1Hive community members</p>
         </StyledGoal>
       </StyledSectionFlex>
-      <StyledSectionFlex wrapSmall={false} style={{ paddingTop: '2rem' }}>
-
-        <MiniCard
-          href="https://1hive.org"
-          target="_blank"
-          rel="noopener noreferer"
-          title={'1Hive DAO Interface'}
-          desc={'Create and stake on Honey distribution proposals'}
-          backgroundColor={'#FF9B73'}
-          color={'black'}
-        />
-        <MiniCard
-          href="https://faucet.1hive.org"
-          target="_blank"
-          rel="noopener noreferer"
-          title={'Honey Faucet'}
-          desc={'Register with BrightId to claim your Honey'}
-          backgroundColor={'#71EEB8'}
-          color={'black'}
-        />
-        <MiniCard
-          href="https://honeyswap.org"
-          target="_blank"
-          rel="noopener noreferer"
-          title={'Honeyswap Interface'}
-          desc={'Trade tokens, add liquidity and create new Honeyswap pools.'}
-          backgroundColor={'#CBF3EF'}
-          color={'black'}
-        />
-        <MiniCard
-          href="https://info.honeyswap.org"
-          target="_blank"
-          rel="noopener noreferer"
-          title={'Honeyswap Info'}
-          desc={'In depth Honeyswap protocol market data.'}
-          backgroundColor={'#FFF19F'}
-          color={'black'}
-        />
-
+       <StyledSectionFlex wrapSmall={false} style={{ paddingTop: '2rem' }}>
+        
+          <MiniCard
+            href="https://1hive.org"
+            target="_blank"
+            rel="noopener noreferer"
+            title={'Honeypot'}
+            desc={'DAO to create and stake on governance proposals.'}
```

Comment:
I would change this back to what it was previously. It's not "a DAO" it's **the** DAO. It's 1hive.

Also, it's not _really_ governance, at least it's not intended to be governance most of the time, it's mostly distribution proposals.

Reply:
I would like to give this one some more consideration.  I feel strongly that we should keep the title as Honeypot to be consistent with our personal names we give each project; honeycomb, honeyswap, etc. Current state we have the statement:
Card - 1Hive DAO Interface - Create and stake on Honey distribution proposals
Menu - 1Hive DAO Interface - Create Proposals and Stake Honey

New Proposal:
Card - Honeypot - The DAO to create and stake on distribution proposals.
Menu - Honeypot - The DAO distribution

Original Proposal:
Card - Honeypot - DAO to create and stake on governance proposals.
Menu - Honeypot - DAO Governance

It would look like this...
![image](https://user-images.githubusercontent.com/73294662/100177194-99350a80-2e9f-11eb-9a70-09ce94dd92f7.png)